Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Login & Signup Flow/NonprofitSignupViewController.m

```diff
@@ -28,8 +28,12 @@ @implementation NonprofitSignupViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    self.nonprofitDescriptionTextView.delegate = self;
     self.finishedSavingBoth = false;
     self.nonprofit = [Nonprofit new];
+    self.nonprofitDescriptionTextView.text = @"Describe your nonprofit in 100 words or less.";
```

Comment:
You should use the `placeholderText` property for this.

Reply:
Hm, I don't think I could find a way to natively set placeholder text with the UITextView, seems like that's only for UITextFields? I might be wrong though

Reply:
Ah, you're right! There's no native `placeholderText` in `UITextView`, just for `UITextField`. I think you should encapsulate and reuse this behavior though, maybe subclass `UITextView` and add the `placeholderText` property to the subclass. You can also use one of the third party components from CocoaPods like [UITextView-Placeholder](https://github.com/devxoul/UITextView-Placeholder).